src/main/java/it/gov/pagopa/payhub/activities/activity/VerifyAuthorizationInstallmentsActivity.java

```diff
@@ -0,0 +1,30 @@
+package it.gov.pagopa.payhub.activities.activity;
+
+import it.gov.pagopa.payhub.activities.dao.OrganizationInstallmentTypeDao;
+import it.gov.pagopa.payhub.activities.dto.InstallmentsOperatorDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationInstallmentTypeDTO;
+import it.gov.pagopa.payhub.activities.exception.custom.ValidatorException;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
```

Comment:
let's write some javadoc, you could reference class methods as an example you could use @{see #verifyAuth(<PARAMETERS, the IDE should support you complete this>)}

`It will handle authorization on DebtPositionType entity`